microsoft-365/admin/manage/use-qr-code-download-outlook.md

```diff
@@ -21,6 +21,10 @@ description: "Learn how to use a QR code to authenticate and download Outlook mo
 
 # Use a QR code to sign-in to the Outlook mobile apps
 
+> [!IMPORTANT]
+> As of August 2021, this experience has been put on HOLD indefinitely on commercial/enterprise users due to the lack of controls on it for Tenants. The works to provide these controls is extensive and the Identity team continues working on them. We rolled back this experience on commercial/enterprise users since we want to provide more security and customization for all the different cases they managed.
```

Comment:
```suggestion
> As of August 2021, this experience has been put on _hold_ indefinitely for commercial and enterprise users due to organizations' lack of control over them. The work involved in providing these controls is extensive and the Identity team continues to work on them. We rolled back this experience on commercial and enterprise users since we wanted to provide more security and customization for all the different cases they managed.
```